app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
+  value: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
+  value: string;
+  max: string;
+};
+
+export class Slider extends React.Component<SliderProps, SliderStates> {
+  state: SliderStates = {
+    value: this.props.value,
+    max: this.props.max,
+  };
+
+  handleChange = (event: any): void => {
+    this.setState({ value: event.target.value });
+  };
+
+  timeFormat = (time: string): string => {
```

Comment:
why previous function is called `handleChange` (verbNoun form) and this one is called `timeFormat` (nounVerb form), shouldn't it be `formatTime`?